src/app/Scenes/InfiniteDiscovery/InfiniteDiscovery.tsx

```diff
@@ -221,6 +165,71 @@ const InfiniteDiscoverySpinner: React.FC = () => (
 
 export const InfiniteDiscoveryQueryRenderer = () => <InfiniteDiscovery />
 
+type InfiniteDiscoveryArtwork = NonNullable<
+  NonNullable<
+    NonNullable<NonNullable<InfiniteDiscoveryQuery$data["discoverArtworks"]>["edges"]>[0]
+  >["node"]
+>
+interface InfiniteDiscoveryArtworkCardProps {
+  artwork: InfiniteDiscoveryArtwork
+}
+
+const InfiniteDiscoveryArtworkCard: React.FC<InfiniteDiscoveryArtworkCardProps> = ({ artwork }) => {
+  const { color } = useTheme()
+  const { width: screenWidth } = useScreenDimensions()
+
+  const src = !!artwork?.images?.[0]?.url ? artwork.images[0].url : undefined
+  const width = !!artwork?.images?.[0]?.width ? artwork.images[0].width : 0
+  const height = !!artwork?.images?.[0]?.height ? artwork.images[0].height : 0
+
+  const size = sizeToFit({ width: width, height: height }, { width: screenWidth, height: 500 })
+
+  return (
+    <Flex backgroundColor={color("white100")} width="100%" height={800}>
+      <EntityHeader
+        name={artwork?.artistNames ?? ""}
+        meta={artwork?.artists?.[0]?.formattedNationalityAndBirthday ?? undefined}
+        imageUrl={artwork?.artists?.[0]?.coverArtwork?.images?.[0]?.url ?? undefined}
+        initials={artwork?.artists?.[0]?.initials ?? undefined}
+        RightButton={
+          <Button variant="outlineGray" size="small">
+            Follow
+          </Button>
+        }
+        p={1}
+      />
+      <Spacer y={2} />
+
+      <Flex alignItems="center" backgroundColor={color("purple60")}>
+        {!!src && <Image src={src} height={size.height} width={size.width} />}
+      </Flex>
+      <Flex flexDirection="row" justifyContent="space-between" p={1}>
+        <Flex>
+          <Flex flexDirection="row" maxWidth={screenWidth - 200}>
+            {/* TODO: maxWidth above and ellipsizeMode + numberOfLines below are used to */}
+            {/* prevent long artwork titles from pushing the save button off of the card, */}
+            {/* it doesn't work as expected on Android. */}
+            <Text
+              color={color("black60")}
```

Comment:
btw, this should work without using `color`
```suggestion
              color={"black60"}
```